<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>379</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>379</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/tb/baidu.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tb_login">
    <attribute name="title">
     <string> 登录 </string>
    </attribute>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>93</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>电话</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pass">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>149</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>93</y>
       <width>142</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>18225041358</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pass">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>149</y>
       <width>142</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_submit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tb_resgester">
    <attribute name="title">
     <string> 注册 </string>
    </attribute>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_submit_register">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pass_register">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_repass_register">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name_register">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name_register">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_pass_register">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>101</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_repass_register">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>44</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>电话</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
